scripts/output/comparison/policyCosts.R

```diff
@@ -233,102 +233,108 @@ happy_with_input <- FALSE
+    message("Type '",crayon::green("y"),"' to continue, '",crayon::blue("r"),"' to reselect output directories, '",crayon::red("n"),"' to abort: ")
+
     user_input <- get_line()
-    
+
     if(user_input %in% c("y","Y","yes")) {
       happy_with_input <- TRUE
-      cat(crayon::green("Great!\n"))
+      message(crayon::green("Great!"))
       
       # Get special requests from user
-      cat(crayon::blue("\nDo you have any special requests?\n"))
-      cat("1: Skip creation of adjustedPolicyCost reporting\n")
-      cat("2: Add transfers to adjustedPolicyCost reporting\n")
-      cat("3: Skip plot creation\n")
-      cat("4: Plot until 2150 in pdf\n")
-      cat("Type the number (or numbers seperated by a comma) to choose the special requests, or nothing to continue without any: ")
-      special_requests <- get_line() %>% str_split(",",simplify = T) %>% as.vector()
-      
-    } else if(user_input %in% c("r","R","reselect")) {
+      message(crayon::blue("\nDo you have any special requests?"))
+      message("1: Skip creation of adjustedPolicyCost reporting")
+      message("2: Add transfers to adjustedPolicyCost reporting")
+      message("3: Skip plot creation")
+      message("4: Plot until 2150 in pdf")
+      message("Type the number (or numbers seperated by a comma) to choose the special requests, or nothing to continue without any: ")
+      special_requests <- get_line() %>% str_split(",",simplify = TRUE) %>% as.vector()
+
+    } else if (user_input %in% c("r","R","reselect")) {
       if (exists("choose_folder")) {
-        cat("Remember, the order in which you choose the directories should be:\n")
-        cat("\t1: policy run 1\n\t2: reference run 1\n\t3: policy run 2\n\t4: reference run 2\nand so on...\n")
-        outputdirs <- choose_folder("./output",crayon::blue("Reselect your ouput folders now"))
+        message("Remember, the order in which you choose the directories should be:")
+        message("\t1: policy run 1\n\t2: reference run 1\n\t3: policy run 2\n\t4: reference run 2\nand so on...")
+        outputdirs <- choose_folder("./output",crayon::blue("Reselect your output folders now"))
         outputdirs <- paste0("output/",outputdirs)
       } else {
-        cat(crayon::red("\nStopping execution now.\n\n"))
+        message(crayon::red("\nStopping execution now.\n"))
         stop("Couldn't find choosefolder() function")
       }
     } else {
-      cat(crayon::red("\nGood-bye (windows xp shutting down music)... \n"))
-      cat(crayon::red("\nStopping execution now.\n\n"))
-      stop("I can't figure this **** out. I give up. ")
+      message(crayon::red("\nGood-bye (windows xp shutting down music)..."))
+      message(crayon::red("Stopping execution now.\n\n"))
+      stop("I can't figure this **** out. I give up.")
     }
   } else {
     happy_with_input <- TRUE
   }
 }
 
 
+message("Copy fulldata.gdx of policy refs to input_refpolicycost.gdx in output folders, eventually overwriting these files.")
```

Comment:
I would say "overwriting these files if they exist" instead of eventually. (Eventually a false-friend ;) )

Reply:
Haha, thanks for spotting that :)